packages/flutter/test/widgets/basic_test.dart

```diff
@@ -1007,6 +1007,63 @@ void main() {
       );
     });
 
+    testWidgets('can merge', (WidgetTester tester) async {
+      final SemanticsTester semantics = SemanticsTester(tester);
+      Widget createWidget() {
+        return Directionality(
+          textDirection: TextDirection.ltr,
+          child: Semantics(
+            label: 'root',
+            container: true,
+            explicitChildNodes: true,
+            child: IgnorePointer(
+              child: Semantics(
```

Comment:
I'm curious, so we merge 3 semantic nodes with inactive tap actions here.  If they all have button flags, should they still be merged?